.travis.yml

```diff
@@ -24,12 +35,9 @@ cache:
   directories:
     - $HOME/.composer/cache
 
-before_install:
-  - travis_retry composer self-update
-
 before_script:
-  - travis_retry composer install --no-interaction
+  - composer update --no-interaction --prefer-source $COMPOSER_FLAGS
```

Comment:
Is there some reason to put `--prefer-source` here? Installing source packages takes more time than dist ones.